icepyx/tests/test_granules.py

```diff
@@ -628,5 +628,5 @@ def test_avail_granule_CMR_error():
     ermsg = "An error was returned from NSIDC in regards to your query: temporal start datetime is invalid: [badinput] is not a valid datetime."
     with pytest.raises(NsidcQueryError, match=re.escape(ermsg)):
         CMRparams = {"version": "003", "temporal": "badinput", "short_name": "ATL08"}
-        reqparams = {"page_size": 1, "page_num": 1}
+        reqparams = {"page_size": 1}
```

Comment:
Is this correct? `page_size = 1`?

Reply:
Yeah - it could be any number, since it's testing for the bad temporal input message and otherwise just needs to be a valid request. Would you recommend using a different number instead?

Reply:
Got it, @JessicaS11 I think you can merge the PR. Have you heard anything from Andy?